hyprbars/barDeco.cpp

```diff
@@ -540,11 +541,36 @@ PHLWINDOW CHyprBar::getOwner() {
     return m_pWindow.lock();
 }
 
-void CHyprBar::setHidden(bool hidden) {
-    if (m_bHidden == hidden)
-        return;
+void CHyprBar::updateRules() {
+    const auto PWINDOW = m_pWindow.lock();
+    auto rules = PWINDOW->m_vMatchedRules;
+    auto prev_m_bHidden = m_bHidden;
+    auto prev_m_bForcedTitleColor = m_bForcedTitleColor;
+
+    m_bForcedBarColor = std::nullopt;
+    m_bForcedTitleColor = std::nullopt;
+    m_bHidden = false;
+
+    for(auto& r : rules) {
+        applyRule(r);
+    }
 
-    m_bHidden = hidden;
+    if (prev_m_bHidden != m_bHidden) {
```

Comment:
no {} around short ifs

Reply:
Thanks, addressed. Do you have a preference regarding short `for`s? The codebase seems to have them both with and without curlies.

Reply:
`for` with {}